src/main/webapp/script.js

```diff
@@ -18,6 +18,29 @@ google.charts.setOnLoadCallback(getChart);
 const CHART_WIDTH = 800;
 const CHART_HEIGHT = 400;
 
+function getYouTubeComments() { 
+  const urlInput = document.getElementById('url-entry');
+  var url = cleanseUrl(urlInput.value);
+  fetch("/YouTubeComments?url="+url)
+      .then(response => response.json()).then((comments) =>{
+        return comments;
+    });
+}
+
+/*
+ * Extracts video id from full url
+ */ 
+function cleanseUrl(url) {
+  // Split web address from parameters, extract first parameter
+  var videoId = url.split("?");
+  videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
+
+  // Remove parameter name to isolate video Id.
```

Comment:
I would update this comment to say: If there is a param name, remove it to isolate video Id.